tests/components/ct-react-vite/playwright/index.tsx

```diff
@@ -2,18 +2,19 @@ import { beforeMount, afterMount } from '@playwright/experimental-ct-react/hooks
 import { BrowserRouter } from 'react-router-dom';
 import '../src/assets/index.css';
 
-export type HooksConfig = {
-  route?: string;
-  routing?: boolean;
+declare module '@playwright/experimental-ct-react/hooks' {
+  interface RegisterHooksConfig {
```

Comment:
We prefer to not override types globally in Playwright. Can we help the user in some other way? Perhaps by casting `test` exported from the package like `const test = baseTest as SomeTestDeclaration<SpecificHooksConfig>`? Or by extending with a `mnt` fixture that is `mount<SpecificHooksConfig>`?

Reply:
> We prefer to not override types globally in Playwright.

Normally i wouldn't use `declare module` in this way, but i think this is a good use case. Especially because the `mount()` is called so many times. The `beforeMount<T>`, `afterMount<T>()` and `mount<T>()` can also still be specified manually for the people who think it's too much magic.

> Can we help the user in some other way?

I don't think there's another way to make the `hooksConfig` in `beforeMount()`, `afterMount()` and `mount()` automatically type-safe with a single type definition right? type-casting `const test` would solve a part of the problem.

Reply:
> I don't think there's another way to make the `hooksConfig` in `beforeMount()`, `afterMount()` and `mount()` automatically type-safe with a single type definition right? type-casting `const test` would solve a part of the problem.

That's right. However, you only write `beforeMount()` and `afterMount()` once, so explicitly typing them is not a big deal. It looks like the main goal is to avoid `mount<T>` every time.